src/Tizen.TV.Extension.UIControls.Forms/Renderer/DRMMediaSourceHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tizen.Multimedia;
+using TM = Tizen.Multimedia;
+using Tizen.TV.Multimedia;
+using TVM = Tizen.TV.Multimedia;
+using Tizen.TV.Extension.UIControls.Forms;
+using Tizen.TV.Extension.UIControls.Forms.Renderer;
+using Tizen.TV.UIControls.Forms.Renderer;
+using Xamarin.Forms;
+
+[assembly: ExportHandler(typeof(DRMMediaSource), typeof(DRMMediaSourceHandler))]
+namespace Tizen.TV.Extension.UIControls.Forms.Renderer
+{
+    public sealed class DRMMediaSourceHandler : IMediaSourceHandler
+    {
+        public Task<bool> SetSource(TM.Player player, TV.UIControls.Forms.MediaSource source)
+        {
+            TVM.Player tvPlayer = player as TVM.Player;
+            if (source is DRMMediaSource uriSource)
+            {
+                var platformDrmMgr = TVM.DRMManager.CreateDRMManager(uriSource.DRMType.ToDRMType());
```

Comment:
Is it a right place to create `DRMManager`?, It will called on every play method call, and it could be called more than one